app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -189,20 +189,23 @@ const externalLinkUnfurlPlugin = async (extension, file) => {
     try {
       metadata = await scrape(file.metadata.resourcePath);
     } catch (e) {
-      console.log(e);
       return file;
     }
     // metadata comes in with properties for each type of unfurl spec (twitter, openGraph etc)
     const combinedMetadata = { ...metadata.twitter, ...metadata.openGraph };
     // const metadata = await metascraper({ html, url });
     // update image to have resource path prepended to it if it is not https
     if (TypeCheck.isString(combinedMetadata.image)) {
-      if (validUrl.isWebUri(combinedMetadata.image) === undefined) {
-        combinedMetadata.image = url.resolve(file.metadata.resourcePath, combinedMetadata.image);
-      }
-
-      file.metadata.unfurl = createUnfurlObj(UNFURL_TYPES.EXTERNAL, combinedMetadata);
+      combinedMetadata.image = url.resolve(file.metadata.resourcePath, combinedMetadata.image);
```

Comment:
`url.resolve` has caused me some grief in the past. If it does not get a URL it fails and in my case it was hard to track down where. It might be worth wrapping it in a try / catch unless you're pretty sure it won't get invalid params.

Reply:
good point,  i didn't think of that. I'll add that try catch. Thanks for the wisdom there

Reply:
actually took a look, I think i should be in the clear. the top level if validates the resource path as a uri